utils/build_swift/driver_arguments.py

```diff
@@ -1045,17 +1045,15 @@ def create_argument_parser():
     option('--tensorflow-host-lib-dir', store_path,
```

Comment:
Technically, this host and target distinction is not needed anymore.

Reply:
Some context: I recall we added the TensorFlow host/target distinction to facilitate future cross-compilation support, at @gribozavr's recommendation.

We mimicked the host/target distinction for some ICU flags (Android-related?) but I can't seem to find those anymore.

Reply:
Yes, but we no longer compile it into swiftc, so the target one is always for cross-compilation.

Reply:
That makes sense!